### ISSUE TITLE
How to obtain iu_ Ray's label path？

### ISSUE THREAD

**Issue by yugui01** [LX-doctorAI1/M2KT#5]

**Comment by FJYfxt** [comment]
Hello, did you solve the problem?

**Comment by kingbeam** [comment]
Hello, did you solve the problem?

**Comment by zddd618** [comment]
How can I get annotation.json in main_basic.py

**Comment by RaySkarken** [comment]
> Hi, can you tell me how to obtain /r2gen_ Split/id_ Is label.csv?
> 
> Can you share the file?
> 
> Thank you!

Try to one hot encode labels from json dataset using pandas.
smth like following code
``` python
import pandas as pd
from sklearn.preprocessing import MultiLabelBinarizer


# Create MultiLabelBinarizer object
mlb = MultiLabelBinarizer()


df = pd.read_json(path_or_buf="all.jsonl", lines=True)
df = df[['id', 'label']]
df['label'] = df['label'].apply(lambda x: [] if x == "''" else list(map(lambda s: s.replace("'", ""), x.split("', '"))))
a = mlb.fit_transform(df['label'])
df = df[['id',]].join(pd.DataFrame(a,
                          columns=mlb.classes_,
                          index=df.index))
```

**Comment by Lalalalala-l** [comment]
> > Hi, can you tell me how to obtain /r2gen_ Split/id_ Is label.csv?
> > Can you share the file?
> > Thank you!
> 
> Try to one hot encode labels from json dataset using pandas. smth like following code
> 
> ```python
> import pandas as pd
> from sklearn.preprocessing import MultiLabelBinarizer
> 
> 
> # Create MultiLabelBinarizer object
> mlb = MultiLabelBinarizer()
> 
> 
> df = pd.read_json(path_or_buf="all.jsonl", lines=True)
> df = df[['id', 'label']]
> df['label'] = df['label'].apply(lambda x: [] if x == "''" else list(map(lambda s: s.replace("'", ""), x.split("', '"))))
> a = mlb.fit_transform(df['label'])
> df = df[['id',]].join(pd.DataFrame(a,
>                           columns=mlb.classes_,
>                           index=df.index))
> ```
can you provide me the annotation.json in main_basic.py,please！

**Comment by Szh2620** [comment]
hello,did you solve this problem? i cant get the label_csv by chexpert
